ipatests/test_integration/test_ntp_replacement.py

```diff
@@ -0,0 +1,151 @@
+
+    @classmethod
+    def install(cls, mh):
+        cls.client = cls.clients[0]
+
+    def _install_client(self, extraargs=[]):
+        cmd = ['ipa-client-install', '-U',
+               '--domain', self.client.domain.name,
+               '--realm', self.client.domain.realm,
+               '-p', self.client.config.admin_name,
+               '-w', self.client.config.admin_password,
+               '--server', self.master.hostname]
+        cmd.extend(extraargs)
+        return self.client.run_command(cmd, raiseonerr=False)
+
+    def test_server_and_client_install_without_option_n(self):
+        """
+        test to verify ipa-server and ipa-client install
+        uses default chrony configuration
+        """
+        expected_msg1 = "No SRV records of NTP servers found and " \
+                        "no NTP server or pool address was provided."
+        expected_msg2 = "Using default chrony configuration."
+        server_install = tasks.install_master(self.master, setup_dns=False)
+        assert expected_msg1 in server_install.stderr_text
+        assert expected_msg2 in server_install.stdout_text
+        client_install = self._install_client()
+        assert expected_msg1 in client_install.stderr_text
+        assert expected_msg2 in client_install.stdout_text
+        self.cleanup()
+
+    def test_server_and_client_install_with_option_n(self):
+        """
+        test to verify ipa-server and ipa-client install with option -N
+        """
+        expected_msg1 = "Excluded by options:"
+        expected_msg2 = "Using default chrony configuration."
+        server_install = tasks.install_master(self.master, setup_dns=False,
+                                              extra_args=['-N'])
+        assert expected_msg1 in server_install.stdout_text
+        assert expected_msg2 not in server_install.stdout_text
+        client_install = self._install_client(extraargs=['-N'])
+        assert expected_msg2 not in client_install.stdout_text
+        self.cleanup()
+
+    def test_server_and_client_install_with_multiple_ntp_server(self):
+        """
+        test to verify ipa-server-install with multiple --ntp-server option
+        """
+        ntp_server1 = "1.pool.ntp.org"
+        ntp_server2 = "2.pool.ntp.org"
+        expected_msg = "Configuration of chrony was changed by installer."
+        server_install = tasks.install_master(self.master,
+                                              setup_dns=False,
+                                              extra_args=[
+                                                  '--ntp-server=%s'
+                                                  % ntp_server1,
+                                                  '--ntp-server=%s'
+                                                  % ntp_server2])
+        assert expected_msg in server_install.stderr_text
+        cmd = self.master.run_command(['cat', paths.CHRONY_CONF])
+        assert "%s" % ntp_server1 in cmd.stdout_text
```

Comment:
drop the string formatting:

``assert ntp_server1 in cmd.stdout_text``

Reply:
done